HW9/hashTable/list.c

```diff
@@ -0,0 +1,102 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include "list.h"
+#define SIZE_OF_LIST 100
+
+typedef struct Node {
+    const char *key;
+    int value;
+    struct Node *left;
+    struct Node *right;
+} Node;
+
+typedef struct List {
+    int size;
+    Node *first;
+    Node *last;
+} List;
+
+List* initList(void) {
+    List *list = calloc(1, sizeof(List));
+    if (list == NULL) {
+        return NULL;
+    }
+    return list;
```

Comment:
```suggestion
    return calloc(1, sizeof(List));
```
Это буквально то же самое.